tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.ServiceTest/appsettings.json

```diff
@@ -130,13 +129,10 @@
     "IsReadOnlyUser": "IsReadOnlyUser",
     "SiteURL": "https://msmatter.sharepoint.com",
     "CentralRepositoryUrl": "https://msmatter.sharepoint.com/sites/catalog",
-    "MailCartMailUserName": "matteradmin@msmatter.onmicrosoft.com",
-    "MailCartMailPassword": "P@$$w0rd01",
     "ExchangeServiceURL": "https://outlook.office365.com/ews/Exchange.asmx"
   },
   "ClientCredentials": {
     "ClientId": "844ffb77-5bfd-403e-9285-678e2eddc90c",
```

Comment:
Do we need ClientId in appsettings prior to connecting to keyvault? if not move this also to keyvault.